crates/cli/commands/src/init_state/mod.rs

```diff
@@ -69,7 +69,7 @@ impl<C: ChainSpecParser<ChainSpec: EthChainSpec + EthereumHardforks>> InitStateC
     where
         N: CliNodeTypes<
             ChainSpec = C::ChainSpec,
-            Primitives: NodePrimitives<BlockHeader: BlockHeader + CliHeader>,
+            Primitives: NodePrimitives<BlockHeader: BlockHeader + HeaderMut>,
```

Comment:
we can collapse this now to just
```suggestion
            Primitives: NodePrimitives<BlockHeader: HeaderMut>,
```